analysis/deep2_r23_o32/plotting/line_ratio_plotting.py

```diff
@@ -11,16 +11,16 @@
 
 def plotting_data1(fitspath, dataset, combine_flux_ascii, bin_info):
     """
-    Purpose
-    Plots measurements to check to make sure that data follows a one-to-one line
-
-    Parameters
-    fitspath -> path where files are called from and saved to
-    dataset  -> keyword used to define binning method
-    combine_flux_ascii -> table of data calculated for emission line
-    bin_info -> table of averages calculated in binning code
+    Purpose:
+      Plots measurements to check to make sure that data follows a one-to-one line
 
+    Parameters:
+      fitspath -> path where files are called from and saved to
+      dataset  -> keyword used to define binning method
+      combine_flux_ascii -> table of data calculated for emission line
+      bin_info -> table of averages calculated in binning code
```

Comment:
Parameters
------------
param: fitspath: str. path where files are called from and saved to
param: dataset: str. keyword used to define binning method
param: combine_flux_ascii: str. table of data calculated for emission line
param: bin_info: str. table of averages calculated in binning code

Reply:
I created #64 for this issue. Will address outside this PR.